src/inference/tests/functional/ov_core_threading.cpp

```diff
@@ -168,17 +169,60 @@ TEST_F(CoreThreadingTests, GetAvailableDevices) {
 }
 
 #if defined(ENABLE_OV_IR_FRONTEND)
+
+namespace ov {
+namespace test {
+namespace util {
+class Barrier {
+private:
+    std::mutex m_mutex;
+    std::condition_variable m_cv;
+    size_t m_count;
+    size_t m_expected;
+    size_t m_gen;
```

Comment:
What does `gen` stand from?

Reply:
Will remove it it looks like for test purposes is not required.